CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 <!-- markdownlint-disable no-duplicate-heading -->
 # Change Log
 
+## 0.8.0
+
+### Minor Changes
+
+* Added support for meta/runtime.yml and execution-environment.yml (#379) @ssbarnea
+* Added schema for ansible-navigator configuration (#365) @ssbarnea
+
+### Bugfixes
+
+* Declared lextudio.restructuredtext ext as conflicting (#366) @ssbarnea
```

Comment:
Writing the changelog for humans means that we should avoid shortening/colloquial speech. This is also emphasized in the style guide that the Ansible tech writers use.
```suggestion
* Declared `lextudio.restructuredtext` extension as conflicting (#366) @ssbarnea
```